### ISSUE TITLE
AppImage crashes on Fedora 37 / Wayland

### ISSUE THREAD

**Issue by amolenaar** [gaphor/gaphor#1920]
### Describe the bug

Starting the latest AppImage on Fedora 37 causes a crash

### To Reproduce
1. Start AppImage from a terminal
2. See it crash:

```
 $ ./Gaphor-2.13.0.dev1937+6761c206-x86_64.AppImage 

(process:596852): GLib-GIO-ERROR **: 13:51:39.573: Settings schema 'org.gnome.settings-daemon.plugins.xsettings' does not contain a key named 'antialiasing'
[1]    596852 trace trap (core dumped)  ./Gaphor-2.13.0.dev1937+6761c206-x86_64.AppImage
```

### OS
- [x] Linux / Wayland
- [ ] macOS
- [ ] Windows

### Version
Version of Gaphor: 2.13.0

### Additional information

On X11 / XWayland it works okay. As a workaround the application can be launched with:

```sh
GDK_BACKEND=x11 ./Gaphor-*.AppImage
```

Not sure if we want to invest time in fixing this. It's a never ending story to get AppImage working on all platforms.

I added a comment on our [Download page](https://gaphor.org/download/).

Not sure if it should be raised as an issue with GTK.

**Comment by danyeaw** [comment]
It works great for me in Wayland on latest OpenSUSE, we'll have to figure out what's unique about Fedora 37.

**Comment by amolenaar** [comment]
Something DBus-y IIRC.

**Comment by danyeaw** [comment]
I just ran Fedora 37 from the Desktop LiveCD, downloaded the AppImage and it launched just fine.

**Comment by danyeaw** [comment]
I am now able to reproduce this on the latest openSUSE with the version 2.14.2 AppImage

**Comment by amolenaar** [comment]
Should be fixed by #2020.
